apps/codeforafrica/src/lib/data/pagify/members.js

```diff
@@ -20,11 +20,22 @@ async function members(api, context) {
     description: member.decription,
     image: member.image,
   };
+  const { docs: relatedProjects } = await api.getCollection("projects", {
```

Comment:
If this function is for a single member, it should be named `member` instead of `members`, right?